paf-mvp-operator-express/tests/integration-tests/operator.integration.test.ts

```diff
@@ -1,4 +1,10 @@
-import { assertError } from '../helpers/integration.helpers';
+import {
+  assertRedirectError,
+  assertRestError,
```

Comment:
Can you take the opportunity (here or in another PR) to remove the warnings of the linter in this file, please?

Reply:
Dismiss my message. I make a proposal here: https://github.com/OneKey-Network/OneKey-implementation/pull/269

Reply:
The warnings here are temporary: the `endMock` variable will be used as soon as we can uncomment the assertions. I didn't think it was worth it to remove it.